sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jdbc/AzureJdbcAutoConfiguration.java

```diff
@@ -26,6 +28,7 @@
 @ConditionalOnBean(DataSourceProperties.class)
 @ConditionalOnClass(AzureAuthenticationTemplate.class)
 @AutoConfigureAfter(DataSourceAutoConfiguration.class)
+@Import(AzureGlobalPropertiesAutoConfiguration.class)
```

Comment:
We'd better not import an AutoConfiguration class from another autoconfiguration class.